chat-with-rendezvous/README.md

```diff
@@ -80,50 +81,53 @@ for _, addr := range bootstrapPeers {
 
 5. **Announce your presence using a rendezvous point.**
 
-[dht.Provide](https://godoc.org/github.com/libp2p/go-libp2p-kad-dht#IpfsDHT.Provide) makes this node announce that it can provide a value for the given key. Where a key in this case is ```rendezvousPoint```. Other peers will hit the same key to find other peers.
+[dht.Provide](https://godoc.org/github.com/libp2p/go-libp2p-kad-dht#IpfsDHT.Provide) makes this node announce that it can provide a value for the given key. Where a key in this case is ```rendezvousString```. Other peers will hit the same key to find other peers. 
 
 ```go
-if err := dht.Provide(tctx, rendezvousPoint, true); err != nil {
-    panic(err)
-}
+routingDiscovery := discovery.NewRoutingDiscovery(kademliaDHT)
+discovery.Advertise(ctx, routingDiscovery, config.RendezvousString)
 ```
 
 6. **Find peers nearby.**
 
 [dht.FindProviders](https://godoc.org/github.com/libp2p/go-libp2p-kad-dht#IpfsDHT.FindProviders) will return all the peers who have announced their presence before.
 
 ```go
-peers, err := dht.FindProviders(tctx, rendezvousPoint)
+peerChan, err := routingDiscovery.FindPeers(ctx, config.RendezvousString)
 ```
 
 **Note:** Although [dht.Provide](https://godoc.org/github.com/libp2p/go-libp2p-kad-dht#IpfsDHT.Provide) and [dht.FindProviders](https://godoc.org/github.com/libp2p/go-libp2p-kad-dht#IpfsDHT.FindProviders) works for a rendezvous peer discovery, this is not the right way of doing it. Libp2p is currently working on an actual rendezvous protocol ([libp2p/specs#56](https://github.com/libp2p/specs/pull/56)) which can be used for bootstrap purposes, real time peer discovery and application specific routing.
```

Comment:
maybe you can say that discovery module uses dht internally

Reply:
Added a line mentioning the internal use, and adding links to those functions